user_groups/user_groups.md

```diff
@@ -298,9 +299,67 @@ Response
 ```
 ---
 
-## Add or update user group permission for a given resource
+## Add user group permission for a given resource
 
-Add or update resource permission of a user group.
+Add resource permission of a user group.
+
+```apacheconfig
+POST /v1/usergroups/{userGroupId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to set up the permission for. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+| Name | Type | Description | Required | Example |
+| ---- | ---- | ------------| :------: | ------- |
+| resourceId | `string` | The resource id to set up the new permission to. | &#9745; | "resourceId": "p-2343" |
+| role | `string` | The user group’s permission role for the resource. | &#9745; | "role": "reader" |
+
+Example
+```json
+{
+  "resourceId": "p-2343",
+  "role": "reader"
+}
+```
+
+#### Response body
+The successful response contains a [Permission Role](../permissions/permissions_schemas.md/#permission-role).
```

Comment:
This is wrong. Successful response contains Permission not PermissionRole.

Reply:
fixed